net-analyzer/bro/bro-2.4.1.ebuild

```diff
@@ -0,0 +1,70 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+PYTHON_COMPAT=( python2_7 python3_{3,4,5} )
+inherit cmake-utils python-single-r1
+
+DESCRIPTION="The Bro Network Security Monitor"
+HOMEPAGE="https://www.bro.org"
+SRC_URI="https://www.bro.org/downloads/release/${PF}.tar.gz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE="+broccoli +broctl broker curl debug geoip ipv6 jemalloc mta +python -ruby tcmalloc static-libs +tools"
+
+RDEPEND="app-shells/bash
+	dev-libs/openssl:0
+	net-analyzer/ipsumdump
+	net-dns/bind-tools
+	net-libs/libpcap
+	sys-libs/zlib
+	broker? ( =dev-libs/actor-framework-0.13.2* )
+	curl? ( net-misc/curl )
+	geoip? ( dev-libs/geoip )
+	jemalloc? ( dev-libs/jemalloc )
+	mta? ( virtual/mta )
+	python? ( ${PYTHON_DEPS} )
+	ruby? ( >=dev-lang/ruby-1.8 )
+	tcmalloc? ( dev-util/google-perftools )"
+DEPEND="${RDEPEND}
+	>=dev-lang/swig-3.0.3
+	dev-lang/perl"
+
+REQUIRED_USE="broker? ( python )
+	broctl? ( python )
+	python? ( ${PYTHON_REQUIRED_USE} )"
+
+src_prepare() {
+	eapply "${FILESDIR}/bro-2.4.1-remove-unnecessary-remove.patch"
+	eapply_user
+}
+
```

Comment:
Since python support is optional in this package, consider overriding python-single-r1's default pkg_setup:

```
pkg_setup() {
	use python && python-single-r1_pkg_setup
}
```